assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -288,10 +290,14 @@ const Checkout = ( {
 									'Shipping options',
 									'woo-gutenberg-products-block'
 								) }
-								description={ __(
-									'Select a shipping method below.',
-									'woo-gutenberg-products-block'
-								) }
+								description={
+									shippingRates.length > 1
```

Comment:
Correct me if I'm wrong, but `shippingRates` is an array of packages. Each package has a `shipping_rates` property which contains the list of options.

So in this case, I think we should check whether there is at least one package with more than one option. Currently, the description is hidden if there is one single package, even if it has more than one shipping option:
![imatge](https://user-images.githubusercontent.com/3616980/78644851-db7ced80-78b6-11ea-994c-94dead2beea6.png)